packages/inventory/src/Stocktake/DetailView/Footer.tsx

```diff
@@ -70,6 +71,15 @@ export const Footer: FC<StocktakeDetailFooterProps> = ({ draft, save }) => {
             alignItems="center"
             height={64}
           >
+            <ToggleButton
+              disabled={!isStocktakeEditable(draft)}
+              value={!!draft.onHold}
+              selected={!!draft.onHold}
+              onClick={() => {
+                draft.updateOnHold();
+              }}
+              label={t('label.hold')}
```

Comment:
In mSupply the label is `Locked` though I prefer having the name consistent and keeping with `Hold`